### PR TITLE
Do not process static methods from Enum class.

### PR DESCRIPTION
**Pull request by xrogers**
Keep original qualified names of the classes.

### CODE REVIEW

File: rules/Php81/Rector/MethodCall/MyCLabsMethodCallToEnumConstRector.php [hunk 1]
```diff
@@ -111,17 +120,19 @@ private function refactorGetValueMethodCall(MethodCall $methodCall): ?PropertyFe
         }
 
         $staticCall = $methodCall->var;
-        $className = $this->getName($staticCall->class);
-        if ($className === null) {
+        $originalName = $staticCall->class->getAttribute(AttributeKey::ORIGINAL_NAME);
```
Comment (TomasVotruba) [inline rules/Php81/Rector/MethodCall/MyCLabsMethodCallToEnumConstRector.php:123]: This should be untouched as other rules always work with fun class names.
Making it short could confuse them and work incorrectly.

Comment (TomasVotruba) [reply rules/Php81/Rector/MethodCall/MyCLabsMethodCallToEnumConstRector.php:123]: This one needs to be resolved before merge.

**Comment by samsonasik** [comment]
There is `importNames()` that should handle use existing use statement:

https://github.com/rectorphp/rector-src/blob/d94353a0acb59823bb74013eb084dd6b302011f5/packages/Config/RectorConfig.php#L72

make short by default by using original name attribute may cause conflict, eg in case of alias usage, or due to virtual node, so I think better to use fqcn as is.

**Comment by TomasVotruba** [comment]
@xrogers I'm just curious, what is your ETA on this PR? :slightly_smiling_face:

**Comment by xrogers** [comment]
@TomasVotruba I've been away, I'll try to finish it this week

**Comment by TomasVotruba** [comment]
Hi @xrogers , let me know if you plan to finish this one or few should move forward and close it. 
I don't want this PR to be opened with no change for next 6 months :)

**Comment by TomasVotruba** [comment]
Closing for lack of feedback, to keep issue tracker focus on unclear issues that needs our attention.
Feel free to send PR with the reproducer :+1:

Thank you for understanding :slightly_smiling_face:

**Comment by TomasVotruba** [comment]
I've cherry picked the test case in https://github.com/rectorphp/rector-src/pull/2911

**Pull request closed without merging**